packer/ansible/roles/node/tasks/main.yml

```diff
@@ -1,15 +1,21 @@
 ---
-- name: Download n Installer
-  get_url: url=http://git.io/n-install dest=/tmp/n-install mode=0755
-  register: download_n
-  sudo: yes
 
-- name: Install n
-  shell: "yes | N_PREFIX={{ ansible_env.HOME }}/n /tmp/n-install"
-  sudo: yes
-  when: download_n|changed
+# source ~/.bashrc is for interactive shell
+- name: Install nvm and Appending source string to ~/.bashrc
+  shell: wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash
```

Comment:
@anhou ah I didn't realize the issues with npm updates and n. Yes, that's a bigger problem, so your decision to use nvm makes sense to me, and we can work out the issues with global installs.